src/backends/torch/torchlib.cc

```diff
@@ -536,6 +537,11 @@ namespace dd
     if (lib_ad.has("loss"))
       _loss = lib_ad.get("loss").get<std::string>();
 
+    if (lib_ad.has("cudnn"))
```

Comment:
Let's not duplicate (used 3 times in the PR) this code and have it into a reusable function ?